src/main/kotlin/com/toasttab/pulseman/pulsar/handlers/KTMessageHandler.kt

```diff
@@ -50,21 +52,27 @@ data class KTMessageHandler(override val cls: Class<out KtMessage>, override val
         val variables = StringBuilder()
         cls.declaredFields
             .filter {
-                it.name !in listOf(
-                    "\$\$delegatedProperties",
-                    "messageSize\$delegate",
-                    "unknownFields",
-                    "Deserializer"
-                )
+                it.name !in IGNORE_FIELDS
             }
             .forEach {
                 if (!it.type.isPrimitive)
                     importSet.add(it.type.name)
-                variables.appendLine("\t${it.name} = //TODO")
+                variables.appendLine("\t${it.name} = //$TODO")
             }
         val imports = StringBuilder()
-        importSet.sorted().forEach { imports.appendLine("import $it") }
+        importSet.sorted().forEach { imports.appendLine("$IMPORT $it") }
 
         return "$imports\n$className {\n$variables}"
```

Comment:
You could change that to a more easily understandable snippet:
```kotlin
"""
$imports
$className {
    $variables
}
""".trimIndent()
```

Reply:
Yeah that looks way nicer, I tried to do it just there but the formattting goes all wonky with the trim indent, I couldnt get it to work easily. Think I'll leave it for now